ThirdPartyAdapters/vungle/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Vungle Android Mediation Adapter Changelog
 
+#### Version 6.7.1.0
+- Verified compatibility with Vungle SDK 6.7.1.
+- Various Bug Fixes
+- Removed Vungle#setUserLegacyID API
+- Added warning flag: On devices <=API 21, OkHttp 3.14+ does not work, use 3.12.12
```

Comment:
Could this cause confusion to readers, we have not introduced this limitation but resolved an issue around it avoiding crash. Suggesting removing this from adapter perspective. We can make sure that IE document it for SDK release